.github/workflows/build.yml

```diff
@@ -45,11 +45,22 @@ jobs:
           echo "$HOME/.sonar/build-wrapper-macosx-x86" >> $GITHUB_PATH
       - name: Run build-wrapper
         run: |
-          build-wrapper-macosx-x86 --out-dir ${{ env.BUILD_WRAPPER_OUT_DIR }} xcodebuild
+          build-wrapper-macosx-x86 --out-dir ${{ env.BUILD_WRAPPER_OUT_DIR }} xcodebuild -project macos-xcode.xcodeproj -scheme macos-xcode
+      - name: Run tests to generate coverage statistics
+        run: |
+          xcodebuild test -project macos-xcode.xcodeproj -scheme test1 -enableCodeCoverage YES -derivedDataPath build
+          xcodebuild test -project macos-xcode.xcodeproj -scheme test2 -enableCodeCoverage YES -derivedDataPath build
+      - name: Collect coverage into one XML report
+        run: |
+          xcrun xcresulttool merge build/Logs/Test/Test-test1-*.xcresult build/Logs/Test/Test-test2-*.xcresult --output-path merged.xcresult
+          bash xccov-to-generic.sh build/Logs/Test/*.xcresult/ > generic-coverage.xml
```

Comment:
I don't understand the data flow here. Based on the steps in README.adoc, "build/Logs/Test/Test-test1-*.xcresult" and "build/Logs/Test/Test-test2-*.xcresult" are used to produce the "merged.xcresult". Is "merged.xresult" used anywhere?
`bash xccov-to-generic.sh` seems to consume again the `.xresult` files from "build/Logs/Test/" so I presume again  "build/Logs/Test/Test-test1-*.xcresult" and "build/Logs/Test/Test-test2-*.xcresult". does it mean that the first step can be skipped?
If not, I think a comment should explain what it is there for.

Reply:
That's a mistake indeed, `merged.xcresult` is not used anywhere.